src/main/java/com/example/businessmanagement2/controller/schedule/ScheduleForm.java

```diff
@@ -17,20 +18,19 @@ public class ScheduleForm {
   int userid;
 
   @DateTimeFormat(pattern = "yyyy-MM-dd")
-  @JsonFormat(pattern = "yyyy/MM/dd")
+  @JsonFormat(pattern = "yyyy-MM-dd")
   @FutureOrPresent
-  Date workingdate;
-
-  String numberoffloors;
+  LocalDate workingdate;
```

Comment:
他にもいくつかありますがキャメルケースが適当かと思います。
```suggestion
  LocalDate workingDate;
```

Reply:
キャメルケースに修正しました！

[42b7657](https://github.com/tomoroltuto/business-management2/pull/4/commits/42b7657f9ad519b6230ff92c2e88e01f601cfdd1)

Reply:
👍